Main.java

```diff
@@ -1,15 +1,28 @@
+import java.util.Scanner;
 /**
- *
- * @author 
+ * determine if a number the user inputs is even or odd
+ * @author Ahmad Zeshan 
  */
 public class Main {
 
   /**
    * The method that is executed when you hit the run button.
    * @param args the command line arguments
    */
-  public static void main(String[] args) {
-    // TODO code application logic here
-    
+  public static void main(String[] args) {
+  // create a Scanner for user input
+	Scanner input = new Scanner(System.in);
+    // prompts the user for an integer
+    System.out.println("Please enter an integer:");
+		//gets the input from user
+		int numberValue = input.nextInt();
+		//divides the number by 2 and if the remiander is 0 it is an even number
+		if (numberValue % 2 == 0){
+		System.out.println(numberValue + " is an even number");
+		}
+		// else it is an odd number
+		else{
+		System.out.println(numberValue + " is an odd number");	
+		}

```

Comment:
The logic here is fine and the commenting is good. Just watch the indentation. So if we take this huge block of code and move it back to the same indentation level as line 16, we would then do an extra indent for anything inside an if statement (21, 25). Otherwise, the code is good!